sdk/zksync.js/package.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "zksync",
-    "version": "0.11.6",
+    "name": "rif-aggregation",
```

Comment:
I know it may be annoying now to change everything, but I'd keep using a more structure format like `@rsksmart/rif-aggregation-sdk-js`

Reply:
@antomor I cannot use that tag name because the package that @jurajpiar publish from the separated [repo](https://github.com/rsksmart/rif-aggregation-sdk-js) is using it.